From c983156c6489e98e4b6986c9636d97e659d0ac11 Mon Sep 17 00:00:00 2001
From: hwg <hwg@rock-chips.com>
Date: Tue, 16 Jun 2015 11:02:56 +0800
Subject: [PATCH] bcmdhd wifi: after android stop; start, wpa_supplicant start
 fail because of Failed to create a P2P Device interface p2p-dev-wlan0

---
 drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/wl_cfgp2p.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/wl_cfgp2p.c b/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/wl_cfgp2p.c
index f8dbbd595ca1..1e3a32aac2e5 100755
--- a/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/wl_cfgp2p.c
+++ b/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/wl_cfgp2p.c
@@ -2795,7 +2795,8 @@ wl_cfgp2p_add_p2p_disc_if(struct bcm_cfg80211 *cfg)
 
 	if (cfg->p2p_wdev) {
 		CFGP2P_ERR(("p2p_wdev defined already.\n"));
-#if (defined(CUSTOMER_HW10) && defined(CONFIG_ARCH_ODIN))
+//#if (defined(CUSTOMER_HW10) && defined(CONFIG_ARCH_ODIN))
+#if 1 // after android stop; start, wpa_supplicant start fail because of "Failed to create a P2P Device interface p2p-dev-wlan0", huweiguo
 		wl_cfgp2p_del_p2p_disc_if(cfg->p2p_wdev, cfg);
 		CFGP2P_ERR(("p2p_wdev deleted.\n"));
 #else
-- 
2.35.3

